src/core/installAddon.js

```diff
@@ -36,14 +36,20 @@ import {
   UNKNOWN,
 } from 'core/constants';
 import * as addonManager from 'core/addonManager';
+import { DISCO_DATA_EXTENSION, DISCO_DATA_THEME } from 'disco/constants';
+
 
 export function installTheme(
   node, addon, { _themeAction = themeAction, _tracking = tracking } = {},
 ) {
   const { name, status, type } = addon;
   if (type === THEME_TYPE && [DISABLED, UNINSTALLED, UNKNOWN].includes(status)) {
     _themeAction(node, THEME_INSTALL);
-    _tracking.sendEvent({ action: 'theme', category: INSTALL_CATEGORY, label: name });
+    _tracking.sendEvent({
```

Comment:
![it-was-like-that-when-i-got-here](https://cloud.githubusercontent.com/assets/90871/21949663/01edde9c-d9ec-11e6-9a81-11624de882df.jpg)

But yes, I'm up for changing it. Also I find it weird we have localised constant files, but that's a separate issue 😄 

I'll change it. 😊